### DISCUSSION TITLE
Change gov token prefix from $ to € to combat imperialism 🇺🇸 and oligarchy 💰 in one fell swoop 🪓☯️

### DISCUSSION THREAD

**Discussion by bmorphism** [DA0-DA0/dao-dao-ui#675] [General]
<img width="420" alt="froggo" src="https://user-images.githubusercontent.com/1236584/164685228-5bf7b258-9c05-4f8c-b602-ff4ad0bcc69b.png">

To align with a dank meme from [ETHPortland 2022](https://2022.ethportland.com/) (aka InterchainPortland 0!),
and pending better mechanisms to deal with purchasable `cw20` governance,
I propose the following simple UI amendment in the meantime:

# 📢📢📢 switch governance token prefix to € to subliminally signal the intended use to be governance 🗳🗳🗳
Why? governance should be earned through a process legitimated over those who choose to delegate it, not bought, lest we fail to shift the status quo, and worse, end up entrenching existing Kapital and improving its dynamism and allocation efficiencies

### Example - [DOG DAO](https://twitter.com/d0gda0)

| Token      | Description | 
|-----------|-------------|
|**[€DOG](https://daodao.zone/dao/juno1hqr0t3scwkrmuu3554lnqzrccuevd8huxmu533gtm6s7vuzeztzqh6tuwq)** |  🇪🇺📜 decide on automation mechanisms (e.g. which dog classifier model to use) for dropping gov|
**[$JUNO](https://www.mintscan.io/juno/account/juno1hqr0t3scwkrmuu3554lnqzrccuevd8huxmu533gtm6s7vuzeztzqh6tuwq)**|  📈🪐 buy food 🌭, hot cocoa / naps, and pay for the dog walker for devs|

### Meta 🧠☯️
at DAO DAO and Interchain more broadly, we have a chance to not only [ape the existing conventions that work](https://github.com/coordinape/coordinape-protocol), but also _set / discover new ones_ that come from our unique focus on and learnings from [governance innovation](https://nickmerrill.substack.com/p/how-governance-could-be)

A few that were discussed in our community calls and are good to keep in view for post-v1:

- blacklists for known DEXes in the contract state itself (ngmi)
- something something curves
- governance power decay (will require a new contract)

One simple (likely too abstruse) UI cue this suggests is to start every token as `€` and to switch to`$` as soon as indexerDAO finds an LP with the gov token attached, or someone reports a secondary market and it is verified in a structured manner.

over and out ☎

**Comment by TrevorJTClarke** [comment]
idk - could just forcefully take-over the © symbol to mean "coin" or "token" 😼

Examples:
©MEOW
©JUNO
©AT
©RAP
©ULT
©LAP

_yes, i am semi-serious_

**Comment by noahsaso** [comment]
Moved this to a discussion, not an issue that needs resolving but an interesting conversation to have.

I also don't love the `$` symbolism, though not sure if we have the social capital to implement this change. If the community likes `$`, we might just have to use it :(
